modules/data.remote/R/MODIS_LAI_prep.R

```diff
@@ -69,20 +69,6 @@ MODIS_LAI_prep <- function(site_info, time_points, outdir = NULL, search_window
   }
   #only Site that has NA for any time points need to be downloaded.
   new_site_info <- site_info %>% purrr::map(function(x)x[!stats::complete.cases(LAI_Output)])
-  #filter out unreachable sites.
-  PEcAn.logger::logger.info("filter out unreachable sites!")
-  non.reachable.ind <- split(as.data.frame(new_site_info), seq(nrow(as.data.frame(new_site_info)))) %>% 
-    furrr::future_map(function(s){
-      if (! "try-error" %in% class(try(mean <- MODISTools::mt_dates(product = "MOD11A2",
-                                                                    lat = s$lat,
-                                                                    lon = s$lon)))) {
-        return(FALSE)
-      } else {
-        return(TRUE)
-      }
-    }, .progress = T) %>% unlist
-  new_site_info <- new_site_info %>% purrr::map(function(x)x[-which(non.reachable.ind)])
-  #if we have any site missing previously
```

Comment:
curious why the code for filtering out unreachable sites as removed

Reply:
The original idea was to make the script more efficient. The NA anchor site run proved that this chunk of code is redundant, as it does not decrease time consumption but increases it. If a site is not reachable, it will end with NA anyway.